### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Disk_DiskToolbar_Keyboard: The user is unable to see any functional changes after the user hits return key on 'Create Snapshot' button using keyboard.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5772]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG2.1.1;#Keyboard;#AINotInScope;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0 
Mac Version: macOS Monterey 12.3.1

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Tab till 'Disks' and select any file.
7. Tab till 'Create Snapshot'
8. Use keyboard to activate.

## Actual:
The user is unable to see any functional changes after the user hits return key on 'Create Snapshot' button using keyboard.

## Expected:
The user should be able to see functional changes after user hits return key on 'Create Snapshot' button using keyboard.

## User Impact:
Keyboard users will be unable to use the feature associated with the control. The user should have been able to create a snapshot in the above mentioned control.

## Attachment:
![31_Create Snapshot](https://user-images.githubusercontent.com/98393966/169970241-a8e95d18-15fb-4711-ab91-d1380afae1fb.gif)

**Comment by jinglouMSFT** [comment]
@bbarik06 "Create Snapshot" should bring up a dialog. Did you see a dialog at all?  If you use mouse instead, do you get a dialog after clicking on "Create Snapshot"?  I am trying to have you verify the functionality first on your machine.

**Comment by bbarik06** [comment]
@jinglouMSFT I tried using mouse to click on 'Create Snapshot' but I don't see any dialog appearing.

**Comment by MRayermannMSFT** [comment]
@bbarik06, can you please:
1. After clicking the "Create Snapshot" button, right click on Storage Explorer in your app dock, and make sure there is not a "Create Snapshot" window in the list: ![image](https://user-images.githubusercontent.com/21112697/170383661-d0f63f58-abc9-4973-92d9-ed3476e3fdcf.png)
If there is, then let us know, but also, please test the dialog after clicking on it.
2. If there is not a "Create Snapshot" window, then please click Help > Open Logs Directory, find the newest folder, right click "Compress", and then drag and drop the resulting zip folder here to GitHub.

Thank you.

**Comment by bbarik06** [comment]
Performed the suggestion (1) and I don't see "Create Snapshot" button and therefore adding the log file for the latest date.
[2022-05-26_160320.zip](https://github.com/microsoft/AzureStorageExplorer/files/8778375/2022-05-26_160320.zip)

**Comment by MRayermannMSFT** [comment]
@bbarik06 how long did you:
- Wait before getting the log files
- Wait before giving up and closing storage explorer?

And, is is possible for you to try this on a Windows machine and see if the same thing happens to you?

**Comment by MRayermannMSFT** [comment]
@bbarik06 no need to do any of that. This is an issue on our end for sure. Will get you a new build soon. Thanks.

**Comment by MRayermannMSFT** [comment]
@bbarik06 can you please try using this build to test the "create disk snapshot" scenario? Let me know if you run into the same issue (nothing happening after clicking the create snapshot button): https://storageexplorerpublish.blob.core.windows.net/privatebuilds/1.24.3-rc-1-build/StorageExplorer-darwin-x64.zip

**Comment by bbarik06** [comment]
@MRayermannMSFT It works now.
<img width="1440" alt="5772_Fixed" src="https://user-images.githubusercontent.com/98393966/175522315-ef8c4304-79dd-4f26-8e27-745861152a70.png">
